pkg/addons/addons.go

```diff
@@ -122,6 +123,11 @@ func enableOrDisableAddon(cc *config.ClusterConfig, name string, val string) err
 		}
 	}
 
+	// to match both ingress and ingress-dns adons
+	if strings.HasPrefix(name, "ingress") && enable && driver.IsKIC(cc.Driver) && runtime.GOOS != "linux" {
+		exit.UsageT("Due to {{.driver_name}} networking limitations on {{.os_name}}, {{.addon_name}} addon is not supported for this driver. meanwhile a different driver such as 'hyperkit' or 'virtualbox' or 'hypev'. To track the update on this feature being worked on please check https://github.com/kubernetes/minikube/issues/7332", out.V{"driver_name": cc.Driver, "os_name": runtime.GOOS, "addon_name": name})
```

Comment:
Simplify this sentence by saying "a VM-based driver" instead of enumerating specific driver. Something like

"Due to {{.driver_name}} networking limitations on {{.os_name}}, the {{.addon_name}} addon is not currently supported. Try using a VM-based driver instead. Follow updates on this feature at https://github.com/kubernetes/minikube/issues/7332."

Reply:
thanks